styleguide/webpack.config.js

```diff
@@ -37,12 +37,6 @@ module.exports = {
                 { collapseGroups: true },
```

Comment:
We might be able to delete more configuration since these might be defaults. I think simplifications > configuration.